ompi/mca/coll/base/coll_base_util.c

```diff
@@ -482,6 +482,30 @@ int ompi_coll_base_file_peek_next_char_is(FILE *fptr, int *fileline, int expecte
     } while (1);
 }
 
+/**
+ * return 1 if the next non-space to read on the current line is a digit.
+ * otherwise return 0.
+ */
+int ompi_coll_base_file_peek_next_char_isdigit(FILE *fptr)
+{
+    do {
+        int next = fgetc(fptr);
+
+        if ((' ' == next) || ('\t' == next)) {
+            continue; /* discard space and tab. keep everything else */
+        }
+
+        ungetc(next, fptr); /* put the char back into the stream */
+
+        if (isdigit(next)) {
```

Comment:
why not `return isdigit(next);` ?

Reply:
It is a good idea. I made the change and pushed it.